README.md

```diff
@@ -1 +1,27 @@
+# csharp-selenium-browserstack
 
+## Prerequisite
+Visual Studio - IDE <br/>
+https://visualstudio.microsoft.com/
+
+Go over browserstack doc to setup visual studio <br/>
+https://www.browserstack.com/docs/automate/selenium/getting-started/c-sharp#introduction
+
+## Steps to run test sessions
+
+1. Configure the capabilites and use your credentials
+(To run a single test, navigate to Single.cs)
+
+```csharp
+capability.AddAdditionalCapability("browser", "iPhone");
```

Comment:
These are sel3 caps, not W3C
Refer https://www.browserstack.com/automate/capabilities?tag=selenium-4